packages/block-library/src/navigation/use-is-collapsed.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import { debounce, useMediaQuery } from '@wordpress/compose';
+import { useState, useLayoutEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { NAVIGATION_MOBILE_COLLAPSE } from './constants';
+import navigationIsWrapping from './is-wrapping';
+
+function useIsCollapsed( overlayMenu, navRef ) {
```

Comment:
Something about this hook is buggy. Its possible to see a wrapped navigation block in the editor.